src/Components/Products/products.css

```diff
@@ -0,0 +1,295 @@
+body {
+  background-color: var(--product-bgcolor);
+  width: 100%;
+  overflow-x: hidden;
+}
+
+.wrapper {
+  overflow-x: hidden;
+}
+
+.navbar {
+  padding: 0;
+  width: 100%;
+}
+
+.nav-link,
+.nav-link:hover {
+  color: var(--bg-color);
+}
+
+.active {
+  color: var(--pink-color);
+}
+
+.product-page {
+  width: 100%;
+  display: flex;
+  margin: 2rem 0;
+  column-gap: 2rem;
+  justify-content: center;
+}
+
+.filter {
+  background-color: var(--white);
+  padding: 2rem;
+  position: sticky;
+  height: 30rem;
+  top: 7rem;
+  left: 0;
+  overflow-y: scroll;
+}
+
+.price-title,
+.category-title,
+.rating-title,
+.sort-by-title {
+  font-size: 1.2rem;
+}
+
+.rating-title {
+  margin-bottom: 1rem;
+}
+
+.sort-by-title {
+  margin: 1rem 0 1rem 0;
+}
+
+.filter-heading {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 1rem;
+  font-size: 1rem;
+}
+
+.cards-list {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  gap: 1.5rem;
+  justify-content: center;
+}
+
+.clear-all-btn {
+  background-color: transparent;
+  border: none;
+  color: var(--pink-color);
+  border-bottom: 1px solid var(--pink-color);
+  cursor: pointer;
+}
+
+.slider,
+.range {
+  width: 12rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.price {
+  margin-bottom: 1rem;
+}
+
+.price-section {
+  margin: 1rem 0;
+}
+
+.filter-category {
+  display: flex;
+  flex-direction: column;
+  row-gap: 0.7rem;
+  cursor: default;
+  margin: 1rem 0;
+}
+
+.category-item-list {
+  display: none;
+}
+
+.category-item:hover .category-item-list {
+  display: block;
+}
+
+.list {
+  display: flex;
+  flex-direction: column;
+  cursor: default;
+  justify-content: flex-start;
+  align-items: flex-start;
+  border: none;
+  font-size: 1rem;
+  background-color: transparent;
+  color: var(--bg-color);
+}
+
+.inline-category-item {
+  display: flex;
+  gap: 0.7rem;
+  justify-content: flex-start;
+  align-items: center;
+  font-size: 1rem;
+  font-weight: 400;
+}
+
+.filter-input {
+  width: 1rem;
+  height: 1rem;
+}
+
+.cat-img {
+  width: 0.9rem;
+  padding-left: 6px;
+}
+
+.product-card {
+  height: fit-content;
+}
+
+.card-padding {
+  padding: 1rem 1rem 0 1rem;
+}
+
+.product-card-title {
+  height: 4rem;
+  font-weight: 450;
+  font-size: 1rem;
+}
+
+.card-btn {
+  width: 83%;
+  border: none;
+}
+
+.no-product {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+@media only screen and (max-width: 64em) {
+  .wrapper {
+    overflow-x: hidden;
+  }
+  .filter {
+    width: 15rem;
+    padding: 10px;
+  }
+  .hide-icon {
+    display: none;
+  }
+  .cards-list {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    gap: 1rem;
+  }
+  .navbar {
+    width: 100%;
+  }
+  .product-card-footer {
+    padding: 0;
+  }
+  .card {
+    width: 12rem;
+    height: fit-content;
+  }
+  .product-page {
+    margin: 1rem;
+  }
+  .card-padding {
+    padding: 12px 12px 0 12px;
+  }
+}
+
+@media only screen and (max-width: 48rem) {
```

Comment:
I think its a typo, this should be in em

Reply:
Done, It was a typo.